### PR TITLE
Initial Integration Request - tomlplusplus

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

**Comment by capuanob** [comment]
I believe this project is security relevant as it is a popular C++ library offered for the parsing of TOML config files (a need of many projects with large user bases). Furthermore, this repository has good visibility as evidenced by its inclusion in the master list of awesome CPP libraries listed at https://github.com/fffaraz/awesome-cpp

Furthermore, the presence of vulnerabilities within the TOML parser could allow a user to achieve DoS by providing faulty config files to applications that use this library. An even worse case would be a malicious actor exploiting a vulnerability to achieve privilege escalation using a higher-elevated process that relies on this library

**Comment by capuanob** [comment]
@jonathanmetzman Good morning, does this submission look good to go? Thank you for your time and your consideration!

**Comment by capuanob** [comment]
@jonathanmetzman I also received approval from the project maintainer as seen [here](https://github.com/marzer/tomlplusplus/issues/208) to support the harnesses and build-scripts upstream

**Comment by capuanob** [comment]
@jonathanmetzman Did the council decide on whether or not tomlplusplus could be accepted? Thank you!

File: projects/tomlplusplus/build.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
```
[the remark this answers is not in the document]
Comment (ennamarie19) [reply projects/tomlplusplus/build.sh]: Removed, my apologies!

**Comment by capuanob** [comment]
@jonathanmetzman Sorry for the delay, I just removed the files that prevented the initial integration request. Thank you for your review!

**Comment by capuanob** [comment]
@Alan32Liu @jonathanmetzman Just pinging to see if this project is good to go, as it has received upstream approval.

Thank you in advance!

**Comment by capuanob** [comment]
@oliverchang Hey Oliver, I was wondering if I could get a status update on this PR request? Thank you in advance!

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**